approve.py

```diff
@@ -0,0 +1,35 @@
+from config import config
+from session import Session
+from datetime import datetime, timezone
+
+def getSavedSearchList(session):
```

Comment:
Function names in Python should be `snake_case`, as is described in PEP 8: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

"Python Enhancement Proposals" are Python's official version of RFCs, and is where the Python community decides on important standards.